### ISSUE TITLE
Major Problem with new topics

### ISSUE THREAD

**Issue by GoogleCodeExporter** [VBGAMER45/prettyurls#88]
```
What steps will reproduce the problem?
1. Pretty URLs (at least) on topic
2. Start new poll
3. Click on add option, or additional options.

What is the expected output? What do you see instead?
More options/extra poll option, 404

What version of the mod, and what version of SMF are you using?
1.0 RC, 1.1.9

What other mods do you have installed?
None that affect that code

Please provide any additional information below.
This has to do with the javascript issues.  Feel free to merge my other
posts here.
```

Original issue reported on code.google.com by `elochaib...@gmail.com` on 9 Jul 2009 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
The javascript issues are already fixed in the patched 1.0RC. Are you using 
that, or
the original 1.0RC?
```

Original comment by `curiousdannii` on 10 Jul 2009 at 2:05

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Most likely was fixed in the 1.0RC patch.
```

Original comment by `curiousdannii` on 21 Jul 2009 at 5:18
- Changed state: **Fixed**
